python_modules/dagster/dagster/_core/run_coordinator/immediately_launch_run_coordinator.py

```diff
@@ -0,0 +1,53 @@
+import logging
+from collections.abc import Mapping
+from typing import Optional
+
+from typing_extensions import Self
+
+import dagster._check as check
+from dagster._config.config_schema import UserConfigSchema
+from dagster._core.run_coordinator.base import RunCoordinator, SubmitRunContext
+from dagster._core.storage.dagster_run import DagsterRun, DagsterRunStatus
+from dagster._serdes import ConfigurableClass, ConfigurableClassData
+
+
+class ImmediatelyLaunchRunCoordinator(RunCoordinator, ConfigurableClass):
```

Comment:
nit: this class name reads a little oddly. Sounds like a sentence not a noun. Perhaps `SynchronousLaunchRunCoordinator` or `ImmediateLaunchRunCoordinator`